test/run_test.py

```diff
@@ -235,7 +235,9 @@
 
 
 # These tests are slow enough that it's worth calculating whether the patch
-# touched any related files first.
+# touched any related files first. This list was manually generated, but for every
+# run with --determine-from, we use another generated list based on this one and the
+# previous test stats.
```

Comment:
might it be useful to provide functionality to print the list generated from S3, or some other way to allow someone to more easily update this manual list?